requirements.prod.in

```diff
@@ -8,4 +8,5 @@ mkdocs-macros-plugin
 mkdocs-material
 https://github.com/opensafely-core/mkdocs-opensafely-databuilder/archive/99f9f3ec5025d5cf9030cafb1189576a4ebf6a7c.zip#mkdocs-opensafely-databuilder
 mkdocstrings[python]
-mkdocs-table-reader-plugin
+https://github.com/rebkwok/mkdocs-table-reader-plugin/archive/298b04ef0f36967a3a3bd24f8325da23885fe945.zip#mkdocs-table-reader-plugin
```

Comment:
suggestion: If we go ahead with using this, it would be good to move this to opensafely-core and/or submit a PR to upstream to add `data_paths` as an option.

Reply:
Also fine to leave it until we decide if it's still needed upon moving out the Data Builder docs.

Reply:
Sure - this was only temporary.  There's a bug for wider use anyway (it can deal with more than one `data_path`, but not if there's more than one used in the same file)